src/board/mod.rs

```diff
@@ -32,14 +32,14 @@ use ruleset::Ruleset;
 use score::Score;
 use self::point::Point;
```

Comment:
My OCD is flaring up when I see that. ;) All use statements together and in alphabetical order, please. Pretty please?